src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -1227,6 +1227,7 @@ public void Deconstruct(out int a, out int b)
             Assert.Equal("(System.Int32 a, System.Int32 b)", tuple2.ToTestDisplayString());
             var underlying2 = tuple2.TupleUnderlyingType;
             Assert.Equal("System.ValueTuple<System.Int32, System.Int32>[missing]", underlying2.ToTestDisplayString());
+            Assert.Equal("(System.Int32 a, System.Int32 b)", model.GetTypeInfo(ab).ConvertedType.ToTestDisplayString());
```

Comment:
>Assert.Equal("(System.Int32 a, System.Int32 b)", model.GetTypeInfo(ab).ConvertedType.ToTestDisplayString()); [](start = 12, length = 108)

Do we have similar test for a regular deconstruction declaration?

Reply:
@AlekseyTs I can't find one. I'll start a PR to add such test. Thanks